insights/parsers/ndctl_list.py

```diff
@@ -0,0 +1,81 @@
+"""
+Commands about "libnvdimm" subsystem devices
```

Comment:
This title is from the manual of `ndctl` command but not `ndctl list` would you please double-check?
- https://www.mankier.com/1/ndctl-list